src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/entity/AlertTest.kt

```diff
@@ -384,6 +384,11 @@ class AlertTest {
         actionedByDisplayName = updatedByDisplayName,
         source = source,
         activeCaseLoadId = activeCaseLoadId,
+        descriptionUpdated = true,
+        authorisedByUpdated = true,
+        activeFromUpdated = true,
+        activeToUpdated = true,
+        commentAppended = true,
```

Comment:
There are quite a few tests in this class that check the domain event has these booleans set properly. The "update alert XXX only" ones. Might be worth also asserting the audit event has the same booleans set.